### PR TITLE
FEATURE: Implemented the random contributor component to display on c…

### LINKED ISSUE #141: [Feature Request] Scroll back to top button
## Feature:
We could create a button that appears into view when the user reaches a certain point down the page. When the user clicks the button it will make the page scroll back to the top. This will let the user quickly go back to the top of the page without having to manually scroll.

## Possible solutions:
Create a button tag with an onClick that takes in a "scrollToTop" function. Inside the scrollToTop function, defined in app.js, you can use the following:
```
window.scrollTo({ top: 0, behavior: smooth })
```
this will scroll the user back to the top.

### Screenshots

_No response_

### Checklist

- [X] I have checked for duplicate [issues](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues)
- [X] I have read the [Contribution Guidelines]((https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md))
- [X] I am willing to work on this issue (optional)

### PR DESCRIPTION
**Pull request by Angeltheesoto**
## Issue Addressed

Closes issue #141

## Proposed Changes and Benefits

- Implemented a scroll to top button that displays when the user reaches 1000 down the page (around 1/3 of view). This will make the user experience more easy for when someone would like to scroll back to the top of the page. Also when the user hovers over the button it scales to 1.1
- Repositioned the random contributors component to be outside of view above the header as well as created a toggle button on top so that when the user clicks, the contributors component will slide into or out of view.

## Screenshots

- Random Contributors component shown at the top of the page.
Screenshot 2023-09-29 140309

- Scroll to top button shown at the bottom right corner of the page.
Screenshot 2023-09-29 140339

### Checklist

- [ ✅ ] I have thoroughly read and understand [The Contribution Guidelines](https://github.com/BeforeIDieCode/BeforeIDieAchievements/blob/main/CONTRIBUTION-GUIDELINES.md).
- [ ✅ ] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
  - `Feature` - adds new or amends existing user-facing behavior.
  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc.
  - `Fix` - bug fixes.
  - `Docs` - fixing typos and adding new content.
- [ ✅ ] I have listed my proposed changes and their benefits.
- [ ✅ ] I have linked the corresponding issue in the Issues section.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -7,8 +7,39 @@ import Footer from "./Components/Footer/Footer";
...
+        <p className="scroll-to-top-arrow">^</p>
+      </div>
+    );
+  };
+
+  useEffect(() => {
+    const handleScroll = () => {
+      const currentPosition = window.scrollY;
+      setShowButton((prev) =>
+        currentPosition < 1000 ? (prev = false) : (prev = true)
```
Comment (XanderRubio) [inline src/App.js:30]: Very neat as you have it set for the scroll button to show on the page when the user has scrolled down more than 1000 pixels from the top. I would agree this is a good approach👏🏻

File: src/Components/randomcontributor/RandomContributors.jsx
Comment (XanderRubio) [inline src/Components/randomcontributor/RandomContributors.jsx:1]: An excellent hide-and-show feature. I like the concept and execution. Looking forward to the design of the component with more UI enhancement. I like how you quickly built out the `isShown` button and added it in the top right corner. I look forward to seeing how this feature continually evolves. You have a good grasp on the useState and useHook to manage state.👏🏻

File: src/css-reset.css
Comment (XanderRubio) [inline src/css-reset.css:1]: Was there a reason each css element is now written one line at a time before it was multiple? Was this from auto format? Just curious.

**Review by XanderRubio — APPROVED** [review]
Well done, @Angeltheesoto! I appreciate the button click feature you added to scroll back up, which functions smoothly on the preview deployment. Also, the show/hide function for the random generator button works well. We will need to find ways to improve the UI to make it fit the website better. Later on we can add the random contributor to the [map page](https://github.com/BeforeIDieCode/BeforeIDieAchievements/issues/24) that is in the process of being worked on, showing where all contributors have contributed on an interactive map. Thank you for your contribution and your comfortability with building in React. I will proceed with merging now!

**Comment by XanderRubio** [comment]
Successfully merged and closes #141

File: src/css-reset.css
Comment (Angeltheesoto) [reply src/css-reset.css:1]: Hi @XanderRubio, I just saw what you meant and yes I believe it was auto-formatted. Sorry about that I'll work on making sure that doesn't happen again next time.

Comment (XanderRubio) [reply src/css-reset.css:1]: All good as I wanted to double-check my understanding of the code👍🏻

**Pull request merged**